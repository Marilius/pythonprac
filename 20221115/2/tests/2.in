class C:
  num = Num()
print(C().num)
c, d = C(), C()
c.num = d.num = 6
print(c.num+d.num)
c.num = "lolkek_test_test"
print(c.num+d.num)
d.num = range(100, 500, 23)
print(c.num+d.num)