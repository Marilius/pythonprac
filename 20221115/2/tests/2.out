0
12
22
34